Daramola98/WEconnectApp
# Creates input validation helpers

#### What does this PR do?
Creates validation helpers for inputs, parameters and queries

#### Description of Task to be completed?
Create method to check business input before it is submitted
Create method to check request parameters if they are valid
Create method to check request queries if they are valid

#### How should this be manually tested?
Clone repository 
Navigate to project root 
Enter command npm start 
Download postman application 
Send post request to endpoint http://localhost/5000/api/v1/weconnect/businesses
Send put request to endpoint http://localhost/5000/api/v1/weconnect/:businesses/:businessId
If any of the inputs passed does not meet the requirements you get a 409 status error and an array containing the error messages
Alternatively you can clone the repository navigate to the root directory and run the command npm test 
All tests should pass
